build.gradle

```diff
@@ -25,7 +25,7 @@ dependencies {
     play "com.typesafe.play:play-guice_$scalaVersion:$playVersion"
     play "com.typesafe.play:play-logback_$scalaVersion:$playVersion"
     play "com.typesafe.play:filters-helpers_$scalaVersion:$playVersion"
-    playTest "org.scalatestplus.play:scalatestplus-play_$scalaVersion:4.0.0-RC2"
+    playTest "org.scalatestplus.play:scalatestplus-play_\$scalaVersion:4.0.0-RC2"
```

Comment:
I'm blocking merge on this PR as a reminder that this replacement is not correct. We should only use this scaping slash on the seeds projects.

Reply:
I resend it. The seed will get broken tough